.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug Report
+about: Bug reports about the Solidity Compiler.
+---
+
+## Prerequisites
+
+- First, many thanks for taking part in the community. We really appreciate that.
+- We realize there is a lot of data requested here. We ask only that you do your best to provide as much information as possible so we can better help you.
+- Support questions are better asked in one of the following locations:
+	- [Solidity chat](https://gitter.im/ethereum/solidity)
+	- [Stack Overflow](https://ethereum.stackexchange.com/)
+- Ensure the issue isn't already reported.
+- Should be reproducible with the latest ``solc`` version, however, this isn't a hard requirement.
```

Comment:
Perhaps start the sentence with `The issue should`?

Reply:
Also `solc` is not what everyone uses, perhaps just use `latest solidity version`